src/cms/views/offers/offer_actions.py

```diff
@@ -37,7 +37,7 @@ def deactivate(request, region_slug, offer_template_slug):
     offer.delete()
     messages.success(
         request,
-        _('Offer "%(offer_name)s" was successfully deactivated.')
+        _('Offer "%(offer_name)s"was successfully deactivated')
```

Comment:
```suggestion
        _('Offer "%(offer_name)s" was successfully deactivated')
```
oh, and here's a space missing :see_no_evil: